compact/definitions.xml

```diff
@@ -583,8 +583,8 @@ Service gaps in FW direction (before endcapP ECAL) and BW direction (before endc
     <constant name="LFHCAL_length"        value="140.0*cm"/>
     <constant name="LFHCAL_zmax"          value="LFHCAL_zmin + LFHCAL_length"/>
 
-    <constant name="HcalEndcapPInsert_zmin"     value="HcalEndcapP_zmin"/>
-    <constant name="HcalEndcapPInsert_length"   value="127.97*cm"/>
+    <constant name="HcalEndcapPInsert_zmin"     value="EcalEndcapP_zmin + EcalEndcapP_length"/>
```

Comment:
Does this imply that EcalEndcapP is to be extended into this space?

Reply:
Ah no, I just changed the zmin from `HcalEndcapP_zmin` to `EcalEndcapP_zmin + EcalEndcapP_length` (which are the same quantity) since the LFHCAL wasn't using `HcalEndcapP_zmin` at all (`LFHCAL_zmin` instead) and I thought this would be clearer. 

I can change it the value of `HcalEndcapPInsert_zmin` to `LFHCAL_zmin` if you think it would be more clear.

Reply:
Oh, my bad, I misread your change to think you replaced `EcalEndcapP_zmin` with `EcalEndcapP_zmin + EcalEndcapP_length`. Of course, there was always `EcalEndcapPInsert` in front, at `EcalEndcapP_zmin`.